src/Symfony/Bundle/FrameworkBundle/Resources/config/mailer.xml

```diff
@@ -39,6 +39,11 @@
             <tag name="kernel.event_subscriber"/>
         </service>
 
+        <service id="mailer.message_listener" class="Symfony\Component\Mailer\EventListener\MessageListener">
+            <argument /> <!-- headers -->
```

Comment:
Can we use an abstract argument here?

Reply:
Sure, but nothing is using it yet in core. So, I prefer to be consistent here. We could have a PR replacing all such usages with abstract argument, but I don't see the see in core.